### ISSUE TITLE
W0221 false positive for property setter

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#30]
Originally reported by: **BitBucket: [udifuchs](http://bitbucket.org/udifuchs)**

---

The following code:

```
#!python

class Foo(object):
    def __init__(self):
        self._bar = 3

    @property
    def bar(self):
        return self._bar

    @bar.setter
    def bar(self, value):
        self._bar = value

class Foo2(Foo):
    @bar.setter
    def bar(self, value):
        self._bar = value * 2

```

Results in the warning:

W0221: 15,4:Foo2.bar: Arguments number differs from overridden method

```
#!sh

$ pylint --version
pylint 0.26.0, 
astng 0.24.1, common 0.58.3
Python 2.7.4 (default, Apr 19 2013, 18:28:01) 
[GCC 4.7.3]

```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/30
